draft-ietf-dnsop-svcb-https.md

```diff
@@ -1014,7 +1018,7 @@ list either.  (Including them wastes space and otherwise has no effect.)
 
 Use of any protocol with SVCB requires a protocol-specific mapping
 specification.  This section specifies the mapping for the "http" and "https"
-URI schemes {{!I-D.draft-ietf-httpbis-semantics}}.
+URI schemes {{!HTTP}}.
```

Comment:
currently we list [HTTP] and [I-D.ietf-httpbis-semantics] as different reference entries.  I don't have a build toolchain set up, though, so I'm not 100% sure this is a complete fix.